packages/hardhat/contracts/strategy/CrabStrategyV2.sol

```diff
@@ -323,95 +361,6 @@ contract CrabStrategyV2 is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Own
-
-        emit TimeHedgeOnUniswap(msg.sender, block.timestamp, auctionTriggerTime, _minWSqueeth, _minEth);
-    }
-
-    /**
-     * @notice hedge startegy based on price threshold with uniswap arbing
-     * @dev this function atomically hedges on uniswap and provides a bounty to the caller based on the difference
-     * @dev between uniswap execution price and the price of the hedging auction
-     * @param _auctionTriggerTime the time when the price deviation threshold was exceeded and when the auction started
-     * @param _minWSqueeth minimum WSqueeth amount the caller willing to receive if hedge auction is selling WSqueeth
-     * @param _minEth minimum ETH amount the caller is willing to receive if hedge auction is buying WSqueeth
-     */
-    function priceHedgeOnUniswap(
-        uint256 _auctionTriggerTime,
-        uint256 _minWSqueeth,
-        uint256 _minEth
-    ) external payable {
-        require(_isPriceHedge(_auctionTriggerTime), "Price hedging not allowed");
-
-        _hedgeOnUniswap(_auctionTriggerTime, _minWSqueeth, _minEth);
-
-        emit PriceHedgeOnUniswap(msg.sender, block.timestamp, _auctionTriggerTime, _minWSqueeth, _minEth);
-    }
-
-    /**
-     * @notice strategy hedging based on time threshold
-     * @dev need to attach msg.value if buying WSqueeth
-     * @param _isStrategySellingWSqueeth sell or buy auction, true for sell auction
-     * @param _limitPrice hedger limit auction price, should be the max price when auction is sell auction, min price when it is a buy auction
-     */
-    function timeHedge(bool _isStrategySellingWSqueeth, uint256 _limitPrice) external payable nonReentrant {
-        (bool isTimeHedgeAllowed, uint256 auctionTriggerTime) = _isTimeHedge();
-
-        require(isTimeHedgeAllowed, "Time hedging is not allowed");
-
-        _hedge(auctionTriggerTime, _isStrategySellingWSqueeth, _limitPrice);
-
-        emit TimeHedge(msg.sender, _isStrategySellingWSqueeth, _limitPrice, auctionTriggerTime);
-    }
-
-    /**
-     * @notice strategy hedging based on price threshold
-     * @dev need to attach msg.value if buying WSqueeth
-     * @param _auctionTriggerTime the time when the price deviation threshold was exceeded and when the auction started
-     * @param _isStrategySellingWSqueeth specify the direction of the trade that you expect, this choice impacts the limit price chosen
-     * @param _limitPrice the min or max price that you will trade at, depending on if buying or selling
-     */
-    function priceHedge(
-        uint256 _auctionTriggerTime,
-        bool _isStrategySellingWSqueeth,
-        uint256 _limitPrice
-    ) external payable nonReentrant {
-        require(_isPriceHedge(_auctionTriggerTime), "Price hedging not allowed");
-        _hedge(_auctionTriggerTime, _isStrategySellingWSqueeth, _limitPrice);
-
-        emit PriceHedge(msg.sender, _isStrategySellingWSqueeth, _limitPrice, _auctionTriggerTime);
-    }
-
-    /**
```

Comment:
we might want re-add these as view functions, so our quoting bot or some f/e or back end can call it to check?